exercises/00_welcome/README.md

```diff
@@ -1,6 +1,6 @@
-# Chapter 0: References and Lifetimes Revision
```

Comment:
Curious if there's a reason to change "Revision" to "Review". Was this confusing to you?

Reply:
Yes, to "revise" something is to make changes to something to improve it. To "review" something in this context is to study again a topic that you've already learned in the past. Here's a good link:

https://www.espressoenglish.net/difference-between-review-and-revise/

Here's some relevant text from that link:
How To Use The Word REVIEW In English
All right, let’s begin with the word review. It has a couple different uses:

One is when you take something you previously learned and you study it again.

Reply:
Ah! That explains it: I speak Australian (i.e. British) English, and:

> However, in British English, the word revise is also used for studying something again. In American English we would always say “review for a text/exam” and in British English you could say “revise for a text/exam.” Again, that use of revise to mean “study again” is specific to British English.

Happy to change it, I try and target American English online. Thanks, and TIL.

Reply:
I wasn't aware of this use of "revise" in British English. I learned something today as well. Cheers!